.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ env:
 
 jobs:
   create_release:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Maybe let's also bump presubmit.yml for consistency? https://github.com/search?q=repo%3Agooglecloudrobotics%2Fcore%20ubuntu-20.04&type=code

I'd also feel comfortable with `ubuntu-latest` if you think that'd reduce work over time.

Reply:
Thanks, went with 22.04 for now, maybe it's nicer not to be surprised when latest changes.